### PR TITLE
[13.0][MIG] product_template_tags

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

**Review by lfreeke — CHANGES_REQUESTED** [review]
This is working but I think it should have a menu option to manage the tags.

**Comment by i-vyshnevska** [comment]
@lfreeke there is menu called  "Product Tags"

**Review by i-vyshnevska — APPROVED** [review]

File: product_template_tags/views/product_template.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='general_information']" position="inside">
```
Comment (emagdalenaC2i) [inline product_template_tags/views/product_template.xml]: See #568

**Comment by lfreeke** [comment]
@i-vyshnevska You are totally correct! I would have expected it somewhere else.

**Review by lfreeke — APPROVED** [review]

File: product_template_tags/views/product_template.xml (hunk 1 above)
Comment (ps-tubtim) [reply product_template_tags/views/product_template.xml]: Thank you. Edited.

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Review by patrickrwilson — APPROVED** [review]
LGTM

**Pull request merged**